SpatialGDK/Documentation/content/tutorials/deployment-manager/tutorial-deploymentmgr-authentication.md

```diff
@@ -39,7 +39,7 @@ _Image: The development authentication token id code as shown in Visual Studio_
 
 If your project opens without errors you have successfully added the developer authentication token to your project. 
 
-**> Next**: [3: Build and upload workers]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers)
+### **> Next**: [3: Build and upload workers]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers)
```

Comment:
Here we have 3 #'s which I think is better. IMO we should have 3 everywhere (this PR adds 1 # to a bunch of the next links)

Reply:
Aah, looks like this was one of those pages that had a different level  of title for the "Next". (I did a regex to make them all `####`, or at least I thought I had.
Will amend.
I agree that `###` is better to spot. :-(

Reply:
```suggestion
#### **> Next**: [3: Build and upload workers]({{urlRoot}}/content/tutorials/deployment-manager/tutorial-deploymentmgr-workers)
```